pkgs/applications/misc/lifelines/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, gettext, libiconv, bison, ncurses, perl, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "lifelines";
+  version = "unstable-2019-05-07";
+
+  src = fetchFromGitHub {
+    owner = "MarcNo";
+    repo = "lifelines";
+    rev = "43f29285ed46fba322b6a14322771626e6b02c59";
+    sha256 = "1agszzlmkxmznpc1xj0vzxkskrcfagfjvqsdyw1yp5yg6bsq272y";
+  };
+
+  buildInputs = [
+    gettext
+    libiconv
+    ncurses
+    perl
+  ];
+  nativeBuildInputs = [ autoreconfHook bison ];
+
+  meta = with stdenv.lib; {
+    description = "Genealogy tool with ncurses interface";
+    homepage = "http://marcno.github.io/lifelines";
```

Comment:
That's a 404
-> maybe https://lifelines.github.io/lifelines/ ? although those pages are outdated

Reply:
@c0bw3b thanks! I had just assumed the values were still valid... I guess that was a bad assumption.